packages/flutter/test/material/text_field_test.dart

```diff
@@ -560,6 +522,8 @@ void main() {
     await tester.pump(const Duration(milliseconds: 50));
 
     await tester.tapAt(textfieldStart + const Offset(150.0, 9.0));
+    // Selection menu renders one frame offstage, so pump twice.
+    await tester.pump();
```

Comment:
I had to add a second pump all over the place due to the selection menu rendering offstage for the first frame.  A single pumpAndSettle didn't work.  Is there any other way to avoid all of these?

Reply:
If it is delayed for a frame, you will have to do this all over